projects/plugins/jetpack/modules/search/class-jetpack-search-customize.php

```diff
@@ -158,7 +158,8 @@ public function customize_register( $wp_customize ) {
 				'type'        => 'select',
 				'choices'     => array(
 					'minimal'  => __( 'Minimal', 'jetpack' ),
-					'expanded' => __( 'Expanded', 'jetpack' ),
+					'expanded' => __( 'Expanded (shows images)', 'jetpack' ),
+					'product'  => __( 'Product grid (for Woo stores)', 'jetpack' ),
```

Comment:
Can we go with `'Product (shows images)'` or `'Product (for WooCommerce stores)'` instead? It's possible that we'll allow for non-grid product views in the future.

Reply:
Good call 👍 I'll change it to `Product (for WooCommerce stores)`.